ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentActionMenu/DocumentActionMenu.js

```diff
@@ -183,7 +184,7 @@ export default class DocumentActionMenu extends React.Component {
           }}
         />
         <NewButton
-          text="New electronic item"
+          text="New E-item"
```

Comment:
Do you prefer the short version? I would actually have left `New electronic item`